easyaudit/signals/model_signals.py

```diff
@@ -67,6 +67,8 @@ def pre_save(sender, instance, raw, using, update_fields, **kwargs):
             if not created:
                 old_model = sender.objects.get(pk=instance.pk)
                 delta = model_delta(old_model, instance)
+                if not delta:
+                    return False
```

Comment:
not sure about returning `False` but seems like a minor thing for now, so it's fine.